java/src/test/java/JavaTest.java

```diff
@@ -1568,13 +1539,6 @@ static void TestPackUnpack(ByteBuffer bb) {
     }
 
     static <T> void TestEq(T a, T b) {
```

Comment:
Since we are adding junit, shouldn't this function be completely removed?

Reply:
Good call, removed and inlined.